parsers/CA_QC.py

```diff
@@ -42,21 +44,19 @@ def if_exists(elem: dict, etype: str):
                 "zoneKey": zone_key,
                 "datetime":  arrow.get(elem["date"], tzinfo=timezone_id).datetime,
                 "production": {
-                    "biomass": 0.0,
+                    "biomass": if_exists(elem, "biomass"),
                     "coal": 0.0,
-
-                    # per https://github.com/tmrowco/electricitymap-contrib/issues/3218 , thermal generation
-                    # is at Bécancour gas turbine. It is reported with a delay, and data source returning 0.0
-                    # can indicate either no generation or not-yet-reported generation.
-                    # To handle this, if reported value is 0.0, overwrite it to None, so that backend can know
-                    # this is not entirely reliable and might be updated later.
-                    "gas": if_exists(elem, "thermal") or None,
-
                     "hydro": if_exists(elem, "hydro"),
                     "nuclear": 0.0,
                     "oil": 0.0,
                     "solar": if_exists(elem, "solar"),
                     "wind": if_exists(elem, "wind"),
+                    # 3218, Québec's thermal generation is at Bécancour gas turbine.
```

Comment:
```suggestion
                    # See Github issue #3218, Québec's thermal generation is at Bécancour gas turbine.
```

Some text got lost here I think, "3218" is starting off the comment on its own